test/integration/deprecated_TEST.cc

```diff
@@ -15,22 +15,18 @@
  *
 */
 
-syntax = "proto3";
-package ignition.msgs;
-option java_package = "com.ignition.msgs";
-option java_outer_classname = "FloatProtos";
+#include <gtest/gtest.h>
 
-/// \ingroup ignition.msgs
-/// \interface Float
-/// \brief A message for float data
+#define SUPPRESS_IGNITION_HEADER_DEPRECATION
 
-import "ignition/msgs/header.proto";
+#include <ignition/msgs/Factory.hh>
+#include <ignition/utils/SuppressWarning.hh>
 
-message Float
+/////////////////////////////////////////////////
+// Make sure the ignition namespace still works
+TEST(Deprecated, IgnitionNamespace)
 {
-  /// \brief Optional header data
-  Header header = 1;
-
-  /// \brief Float data
-  float data = 2;
+  ignition::msgs::Factory factory;
```

Comment:
> Oh yeah, and because so many files are being moved, we'll need a clean commit history to rebase-merge here. upside_down_face

The files being moved here were the redirection headers added in the header migration, do we need to preserve the history for those?